docs/user-guide/administration/manage-etcd.md

```diff
@@ -0,0 +1,228 @@
+---
+keywords: [etcd]
+description: a etcd management documentation.
+---
+
+# Manage ETCD
+
```

Comment:
please add an overview to describe: what is etcd, why developers need to manage etcd